protobuf.bzl

```diff
@@ -400,3 +400,10 @@ def internal_protobuf_py_tests(
         srcs=[s],
         main=s,
         **kargs)
+
+
+def check_version():
```

Comment:
It seems better to specify the required minimum version number in the BUILD file instead of here. How about change this to:

```
def require_bazel_version(version):
  ...
```

And in the BUILD file do:

```
load("@com_google_protobuf//:protobuf.bzl", "require_bazel_version")
require_bazel_version("0.5.4")
```

Reply:
I wish, but bazel doesn't expose its version in the BUILD rules, only in the WORKSPACE

Reply:
Can you change this to `require_bazel_version()` and do `require_bazel_version("0.5.4")` in the toplevel WORKSPACE file?

Reply:
That WORKSPACE file doesn't affect users at all, right? Like if in my WORKSPACE I pull in com_google_protobuf, bazel never looks at your WORKSPACE file

Reply:
Really? I think bazel has to look at protobuf's WORKSPACE file because otherwise it won't know what library protobuf depends on and won't be able to build protobuf.

Reply:
I wish, I have to have the `python_header` bind in my WORKSPACE too. bazel only looks at the top-level WORKSPACE and all loaded BUILD and .bzl files. Plenty of other repositories do something like this:

```py
git_repository(
    name = "io_bazel_rules_pex",
    remote = "https://github.com/fahhem/bazel_rules_pex.git",
    tag = "applied-1.0",
)
load("@io_bazel_rules_pex//pex:pex_rules.bzl", "pex_repositories")
pex_repositories()
```

That way they put all their dependencies into a bzl file, and include that in their own WORKSPACE and all users' WORKSPACEs

Reply:
Can you change the name of this function to `check_protobuf_required_bazel_version`?

Reply:
Also please add comment for this function to explain how this function is supposed to be used.

Reply:
Done, thanks! PTAL